Tests/MongoSwiftTests/MongoClientTests.swift

```diff
@@ -165,4 +166,41 @@ final class MongoClientTests: MongoSwiftTestCase {
         // wait to ensure all resource cleanup happens correctly
         try closeFuture.wait()
     }
+
+    func testOCSP() throws {
+        guard ProcessInfo.processInfo.environment["MONGODB_OCSP_TESTING"] != nil else {
+            printSkipMessage(testName: "testOCSP", reason: "MONGODB_OCSP_TESTING environment variable not set")
+            return
+        }
+
+        guard
+            let shouldSucceed = ProcessInfo.processInfo.environment["OCSP_TLS_SHOULD_SUCCEED"].flatMap(Bool.init)
+        else {
+            throw MongoError.InternalError(message: "OCSP_TLS_SHOULD_SUCCEED not set or has invalid value")
+        }
+
+        var options = MongoClientOptions(serverSelectionTimeoutMS: 200)
+        try self.withTestClient(options: options) { client in
+            let response = Result {
+                try client.db("admin").runCommand(["ping": 1]).wait()
+            }
+
+            if shouldSucceed {
+                expect(try response.get()).toNot(throwError())
+            } else {
+                expect(try response.get()).to(throwError())
```

Comment:
nit: should we assert the error is of an expected type here?

Reply:
done. It should just time out on server selection since the client can't discover any servers successfully.